### DISCUSSION TITLE
How to percept connectivity/bondtype/rings using c++?

### DISCUSSION THREAD

**Discussion by zhangfq-chemistry** [rdkit/rdkit#7609] [General]
How to  load xyz file (no bond information available), then determine connectivity/bondtype/rings using c++?

**Comment by greglandrum** [comment]
There's a blog post with an intro here: https://greglandrum.github.io/rdkit-blog/posts/2022-12-18-introducing-rdDetermineBonds.html

That should give and overview and let you know what search terms to use if you want more info
